quantum/split_common/transport.c

```diff
@@ -209,8 +220,14 @@ bool transport_master(matrix_row_t matrix[]) {
     }
 #    else
     transport_rgblight_master();
-    if (soft_serial_transaction(GET_SLAVE_MATRIX) != TRANSACTION_END) {
-        return false;
+
+    static uint8_t s_last_slave_buffer_change_count = 0;
+    if (soft_serial_transaction(GET_SLAVE_STATUS) == TRANSACTION_END && slave_buffer_change_count != s_last_slave_buffer_change_count) {
+        printf("matrix sync needed:%u:%u\n", slave_buffer_change_count, s_last_slave_buffer_change_count);
```

Comment:
Lets remove the printf as it's causing CI failures.

Reply:
Also, dprintf would work better here, I think